### ISSUE TITLE
Tensorboard Learning rate has disappeared after the update!

### ISSUE THREAD

**Issue by JangJooYoungSNU** [tensorflow/models#3747]
Please go to Stack Overflow for help and support:

http://stackoverflow.com/questions/tagged/tensorflow

Also, please understand that many of the models included in this repository are experimental and research-style code. If you open a GitHub issue, here is our policy:

1. It must be a bug, a feature request, or a significant problem with documentation (for small docs fixes please send a PR instead).
2. The form below must be filled out.

**Here's why we have that policy**: TensorFlow developers respond to issues. We want to focus on work that benefits the whole community, e.g., fixing bugs and adding features. Support only helps individuals. GitHub also notifies thousands of people when issues are filed. We want them to see you communicating an interesting problem, rather than being redirected to Stack Overflow.

------------------------

### System information
- **What is the top-level directory of the model you are using**: **models/research**
- **Have I written custom code (as opposed to using a stock example script provided in TensorFlow)**:no
- **OS Platform and Distribution (e.g., Linux Ubuntu 16.04)**: Linux Ubuntu 16.04
- **TensorFlow installed from (source or binary)**: Tensorflow-gpu source
- **TensorFlow version (use command below)**: 1.6.0
- **Bazel version (if compiling from source)**:not used
- **CUDA/cuDNN version**: 9.0.176 / 7.0
- **GPU model and memory**: Titan XP
- **Exact command to reproduce**: tensorboard --logdir=path/to/events.out.* --inspect

You can collect some of this information using our environment capture script:

https://github.com/tensorflow/tensorflow/tree/master/tools/tf_env_collect.sh

You can obtain the TensorFlow version with

python -c "import tensorflow as tf; print(tf.GIT_VERSION, tf.VERSION)"

### Describe the problem
Describe the problem clearly here. Be sure to convey here why it's a bug in TensorFlow or a feature request.
Hello, my problem is that whenever I open up tensorboard, I couldn't find learning rate scalar value in my tensorboard after I updated the object detection api a month ago. (Feburary)

![image](https://user-images.githubusercontent.com/13467628/37894976-f150606e-311a-11e8-8231-c86f51ffa6df.png)

<before>

![image](https://user-images.githubusercontent.com/13467628/37894971-eaec77da-311a-11e8-99d0-1acd14429b4b.png)

<after>

as you can see in the pictures above, learning rate has been disappeared!! T.T
Can somebody tell me where should I change the code ??
I searched for models/research/object_detection/trainer.py part (specially, tf.add.summary and tf.scalar) but, there was no difference between after / before.

Thanks in advance!
### Source code / logs
Include any logs or source code that would be helpful to diagnose the problem. If including tracebacks, please include the full traceback. Large logs and files should be attached. Try to provide a reproducible test case that is the bare minimum necessary to generate the problem.

**Comment by yangsiyu007** [comment]
Also missing learning rate and it would be really helpful to log it.

**Comment by ymodak** [comment]
Is this still an issue?

**Comment by ymodak** [comment]
Automatically closing due to lack of recent activity. Please update the issue when new information becomes available, and we will reopen the issue. Thanks!
